docs/secure-connections/sauce-connect/installation.md

```diff
@@ -70,6 +66,19 @@ tar xvf ./sc-4.7.1-linux.tar.gz
 export PATH="$HOME/sc-4.7.1-linux/bin:$PATH"
 ```
 
+
+## Using Docker Containers to Install/Run Sauce Connect
+
+You can use the Sauce Connect Proxy Docker image, maintained by the Sauce Labs [Open Source Program Office](https://opensource.saucelabs.com/), to build a Docker container. This means it is an alternative to the above Download/Installation steps.
+
+If you’re not sure where Docker fits, consider these use cases:
+* If you want to creating automation (or manual testing) for multiple apps that reside in different environments or Sauce Labs Data Centers (i.e., US-East and US-West).
+* If you want to create multiple High Availability Sauce Connect tunnels on the same system, without having to manage or use different ports.
+* If you no longer want to manage Sauce Connect Proxy versions.
+* If you want to run Sauce Connect Proxy as part of a dockerized CI.
```

Comment:
I'd change that to smth like:

- If you run Sauce Connect Proxy as part of a dockerized CI
- If you prefer to manage docker image tags instead of Sauce Connect Proxy versions
- If your setup involves several  instances running on the same system, docker would simplify Sauce Connect Proxy port management